src/matching/Matching_generic.ml

```diff
@@ -114,6 +114,7 @@ type tin = {
   lang : Lang.t;
   config : Rule_options.t;
   deref_sym_vals : int;
+  wildcard_imports : AST_generic.ident list list;
```

Comment:
`AST_generic.dotted_ident list`

Reply:
(We should probably have a type alias for `dotted_ident` with an unboxing annotation...)